### PR TITLE
Add Book Dash reports from 2021 and 2022

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

Add all the book dash reports

### List of changes proposed in this PR (pull-request)

* [x] May 2021
* [x] Nov 2021
* [x] May 2022
* [x] Nov 2022
* [x] Rename other files to keep them in chronological order

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

File: workshops/book-dash/2021-May-BookDash-5.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+* Document with all links: https://hackmd.io/@turingway/bookdash-may2021-index
+* For specific notes from each day, please visit one of the following notes:
+    * **Day 1, 17 May**: https://hackmd.io/@turingway/bookdash-may2021-day1
+    * **Day 2, 18 May**: https://hackmd.io/@turingway/bookdash-may2021-day2
+    * **Day 3, 19 May**: https://hackmd.io/@turingway/bookdash-may2021-day3
+    * **Day 4, 20 May**: https://hackmd.io/@turingway/bookdash-may2021-day4
+    * **Day 5, 21 May**: https://hackmd.io/@turingway/bookdash-may2021-day5
+* Reports from the previous events:
+    * 
+    * https://github.com/alan-turing-institute/the-turing-way/tree/master/workshops/book-dash
```
Comment (Arielle-Bennett) [inline workshops/book-dash/2021-May-BookDash-5.md:17]:
```suggestion
    * https://github.com/alan-turing-institute/the-turing-way/tree/main/workshops/book-dash
```

File: workshops/book-dash/2022-May-BookDash-7.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+* Friendly environment, I felt able/encouraged to try out different things and experiment a bit, while at the same time I did accomplish something!
+* The hub was very nice: it was really great to sit and work together and meet others in the community I hadn't met before.
+* Liked the reminders to take breaks but having the option to have people to hold me accountable
+* 
+
+Delta
+---
+
+* Some of the templates were inconsistent with timings and that caused confusion 
+* A way to keep track of all the different documents. Maybe one master HackMD/document which links to all the other HackMDs or important Slack info? (Like a password manager) So we can find the other documents or illustrations easily
```
Comment (Arielle-Bennett) [inline workshops/book-dash/2022-May-BookDash-7.md:107]:
```suggestion
* A way to keep track of all the different documents. Maybe one central HackMD/document which links to all the other HackMDs or important Slack info? (Like a password manager) So we can find the other documents or illustrations easily
```

File: workshops/book-dash/2022-May-BookDash-7.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+### Still wondering where to start?
+
+Here are a few suggestions for good first contributions:
+
+-   Fix a typo in [the book](https://the-turing-way.netlify.com) and/or [repository](https://github.com/alan-turing-institute/the-turing-way).
+-   Flag the unclear or duplicated information in the book by opening an [issue on the GitHub](https://github.com/alan-turing-institute/the-turing-way/issues).
+-   Fix an [old issue](https://github.com/alan-turing-institute/the-turing-way/issues) or comment on the the [pull requests](https://github.com/alan-turing-institute/the-turing-way/pulls).
+-   Write your personal experience of reproducible research in this [google form](https://goo.gl/forms/akFqZEIy2kxAjfZW2).
+-   See more info on the GitHub: [https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/online-collaboration-cafe.md](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/online-collaboration-cafe.md)
```
Comment (Arielle-Bennett) [inline workshops/book-dash/2022-May-BookDash-7.md:134]:
```suggestion
-   See more info on the GitHub: [https://github.com/alan-turing-institute/the-turing-way/blob/main/book/website/community-handbook/coworking/coworking-collabcafe.md](https://github.com/alan-turing-institute/the-turing-way/blob/main/book/website/community-handbook/coworking/coworking-collabcafe.md)
```

File: workshops/book-dash/2021-Nov-BookDash-6.md [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+
+### Still wondering where to start?
+
+Here are a few suggestions for good first contributions:
+
+-   Fix a typo in [the book](https://the-turing-way.netlify.com) and/or [repository](https://github.com/alan-turing-institute/the-turing-way).
+-   Flag the unclear or duplicated information in the book by opening an [issue on the GitHub](https://github.com/alan-turing-institute/the-turing-way/issues).
+-   Fix an [old issue](https://github.com/alan-turing-institute/the-turing-way/issues) or comment on the the [pull requests](https://github.com/alan-turing-institute/the-turing-way/pulls).
+-   Write your personal experience of reproducible research in this [google form](https://goo.gl/forms/akFqZEIy2kxAjfZW2).
+-   See more info on the GitHub: [https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/online-collaboration-cafe.md](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/online-collaboration-cafe.md)
```
Comment (Arielle-Bennett) [inline workshops/book-dash/2021-Nov-BookDash-6.md:162]:
```suggestion
-   See more info on the GitHub: [https://github.com/alan-turing-institute/the-turing-way/blob/main/book/website/community-handbook/coworking/coworking-collabcafe.md](https://github.com/alan-turing-institute/the-turing-way/blob/main/book/website/community-handbook/coworking/coworking-collabcafe.md)
```

File: workshops/book-dash/2022-May-BookDash-7.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+* Scriberia Session Bookings: https://hackmd.io/@turingway/bookdash-nov2021-scriberia
+* For specific notes from each day, please visit one of the following notes:
+    * **Day 1, 08 November**: https://hackmd.io/@turingway/bookdash-nov2021-day1
+    * **Day 2, 09 November**: https://hackmd.io/@turingway/bookdash-nov2021-day2
+    * **Day 3, 10 November**: https://hackmd.io/@turingway/bookdash-nov2021-day3
+    * **Day 4, 11 November**: https://hackmd.io/@turingway/bookdash-nov2021-day4
+    * **Day 5, 12 November**: https://hackmd.io/@turingway/bookdash-nov2021-day5
+* Post-event feedback: https://hackmd.io/@turingway/bookdash-nov2021-feedback
+* Reports from the previous events:
+    * https://github.com/alan-turing-institute/the-turing-way/tree/master/workshops/book-dash
```
Comment (Arielle-Bennett) [inline workshops/book-dash/2022-May-BookDash-7.md:154]:
```suggestion
    * https://github.com/alan-turing-institute/the-turing-way/tree/main/workshops/book-dash
```

File: workshops/book-dash/2022-Nov-BookDash-8.md [hunk 6]
```diff
@@ -0,0 +1,225 @@
...
+
+### Still wondering where to start?
+
+Here are a few suggestions for good first contributions:
+
+-   Fix a typo in [the book](https://the-turing-way.netlify.com) and/or [repository](https://github.com/alan-turing-institute/the-turing-way).
+-   Flag the unclear or duplicated information in the book by opening an [issue on GitHub](https://github.com/alan-turing-institute/the-turing-way/issues).
+-   Fix an [old issue](https://github.com/alan-turing-institute/the-turing-way/issues) or comment on the the [pull requests](https://github.com/alan-turing-institute/the-turing-way/pulls).
+-   Write your personal experience of reproducible research in this [google form](https://goo.gl/forms/akFqZEIy2kxAjfZW2).
+-   See more info on GitHub: [https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/online-collaboration-cafe.md](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/online-collaboration-cafe.md)
```
Comment (Arielle-Bennett) [inline workshops/book-dash/2022-Nov-BookDash-8.md:199]:
```suggestion
-   See more info on GitHub: [https://github.com/alan-turing-institute/the-turing-way/blob/main/project_management/online-collaboration-cafe.md](https://github.com/alan-turing-institute/the-turing-way/blob/main/project_management/online-collaboration-cafe.md)
```

File: workshops/book-dash/organisers_files-2021/2021-may-index.md [hunk 7]
```diff
@@ -0,0 +1,265 @@
...
+
+### GitHub Resources
+
+- GitHub repository: https://github.com/alan-turing-institute/the-turing-way
+- Online Book: https://the-turing-way.netlify.app/welcome
+- Impact story: https://www.turing.ac.uk/research/impact-stories/changing-culture-data-science
+    - [Short summary](https://www.turing.ac.uk/sites/default/files/inline-images/Turing%20Way%20oner_0.jpg)
+- Open issues: https://github.com/alan-turing-institute/the-turing-way/issues
+- Open Pull Requests: https://github.com/alan-turing-institute/the-turing-way/pulls
+- [Reports from the previous Book Dashes](https://github.com/alan-turing-institute/the-turing-way/tree/master/workshops/book-dash)
```
Comment (Arielle-Bennett) [inline workshops/book-dash/organisers_files-2021/2021-may-index.md:193]:
```suggestion
- [Reports from the previous Book Dashes](https://github.com/alan-turing-institute/the-turing-way/tree/main/workshops/book-dash)
```

File: workshops/book-dash/organisers_files-2021/2021-may-index.md [hunk 8]
```diff
@@ -0,0 +1,265 @@
...
+
+The tl;dr version for easy reference during dashing:
+1. Comment on an [existing issue](https://github.com/alan-turing-institute/the-turing-way/issues) or open a new issue referencing your addition.
+    * The [issue template](https://github.com/alan-turing-institute/the-turing-way/blob/master/ISSUE_TEMPLATE.md) will automatically be rendered in the comment section of the new issue page, so all you need to do is edit the "*Lorem ipsum*" sections.
+    * Add the ```book-dash-<location>``` label in the column on the right.
+2. [Fork](https://help.github.com/articles/fork-a-repo) the Turing Way repository to your GitHub profile.
+3. Make your changes!
+4. Submit a [pull request](https://help.github.com/articles/creating-a-pull-request).
+    * Open a pull request as early in your contributing process as possible and add the label [WIP] to the title to designate "work in progress".
+    * The [pull request template](https://github.com/alan-turing-institute/the-turing-way/blob/master/.github/PULL_REQUEST_TEMPLATE.md) will automatically be rendered in the comment section of the new pull request page, so all you need to do is edit the "*Lorem ipsum*" sections.
```
Comment (Arielle-Bennett) [inline workshops/book-dash/organisers_files-2021/2021-may-index.md:211]:
```suggestion
    * The [pull request template](https://github.com/alan-turing-institute/the-turing-way/blob/main/.github/PULL_REQUEST_TEMPLATE.md) will automatically be rendered in the comment section of the new pull request page, so all you need to do is edit the "*Lorem ipsum*" sections.
```

File: workshops/book-dash/organisers_files-2021/2021-nov-index.md [hunk 9]
```diff
@@ -0,0 +1,269 @@
...
+
+**Please read and follow our project [Contributing Guidelines](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md)!** :sparkles: 
+
+If you find them confusing, difficult to follow or think that information needs to be added, please let us know! This will help us improve the contributing experience for everyone going forward.
+
+### Managing your contributions to the GitHub repository
+
+The tl;dr version for easy reference during dashing:
+1. Comment on an [existing issue](https://github.com/alan-turing-institute/the-turing-way/issues) or open a new issue referencing your addition.
+    * The [issue template](https://github.com/alan-turing-institute/the-turing-way/blob/master/ISSUE_TEMPLATE.md) will automatically be rendered in the comment section of the new issue page, so all you need to do is edit the "*Lorem ipsum*" sections.
```
Comment (Arielle-Bennett) [inline workshops/book-dash/organisers_files-2021/2021-nov-index.md:209]:
```suggestion
    * The [issue template](https://github.com/alan-turing-institute/the-turing-way/blob/main/ISSUE_TEMPLATE.md) will automatically be rendered in the comment section of the new issue page, so all you need to do is edit the "*Lorem ipsum*" sections.
```

File: workshops/book-dash/organisers_files-2022/2022-may-index.md [hunk 10]
```diff
@@ -0,0 +1,275 @@
...
+
+Any day-to-day feedback during the event can be directly shared in the main event document.
+
+:books: _The Turing Way_ links and resources
+---
+
+### Participation guidelines
+
+- Code of Conduct: https://the-turing-way.netlify.app/community-handbook/coc.html
+- Contribution guideline: https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md
```
Comment (Arielle-Bennett) [inline workshops/book-dash/organisers_files-2022/2022-may-index.md:176]:
```suggestion
- Contribution guideline: https://github.com/alan-turing-institute/the-turing-way/blob/main/CONTRIBUTING.md
```

File: workshops/book-dash/organisers_files-2022/2022-may-index.md [hunk 11]
```diff
@@ -0,0 +1,275 @@
...
+
+### GitHub Resources
+
+- GitHub repository: https://github.com/alan-turing-institute/the-turing-way
+- Online Book: https://the-turing-way.netlify.app/welcome
+- Impact story: https://www.turing.ac.uk/research/impact-stories/changing-culture-data-science
+    - [Short summary](https://www.turing.ac.uk/sites/default/files/inline-images/Turing%20Way%20oner_0.jpg)
+- Open issues: https://github.com/alan-turing-institute/the-turing-way/issues
+- Open Pull Requests: https://github.com/alan-turing-institute/the-turing-way/pulls
+- [Reports from the previous Book Dashes](https://github.com/alan-turing-institute/the-turing-way/tree/master/workshops/book-dash)
```
Comment (Arielle-Bennett) [inline workshops/book-dash/organisers_files-2022/2022-may-index.md:198]:
```suggestion
- [Reports from the previous Book Dashes](https://github.com/alan-turing-institute/the-turing-way/tree/main/workshops/book-dash)
```

File: workshops/book-dash/organisers_files-2022/2022-may-index.md [hunk 12]
```diff
@@ -0,0 +1,275 @@
...
+
+## :gift: Your contribution to the project
+
+**Please read and follow our project [Contributing Guidelines](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md)!** :sparkles:
```
Comment (Arielle-Bennett) [inline workshops/book-dash/organisers_files-2022/2022-may-index.md:202]:
```suggestion
**Please read and follow our project [Contributing Guidelines](https://github.com/alan-turing-institute/the-turing-way/blob/main/CONTRIBUTING.md)!** :sparkles: 
```

File: workshops/book-dash/organisers_files-2022/2022-may-index.md [hunk 13]
```diff
@@ -0,0 +1,275 @@
...
+
+**Please read and follow our project [Contributing Guidelines](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md)!** :sparkles: 
+
+If you find them confusing, difficult to follow or think that information needs to be added, please let us know! This will help us improve the contributing experience for everyone going forward.
+
+### Managing your contributions to the GitHub repository
+
+The tl;dr version for easy reference during dashing:
+1. Comment on an [existing issue](https://github.com/alan-turing-institute/the-turing-way/issues) or open a new issue referencing your addition.
+    * The [issue template](https://github.com/alan-turing-institute/the-turing-way/blob/master/ISSUE_TEMPLATE.md) will automatically be rendered in the comment section of the new issue page, so all you need to do is edit the "*Lorem ipsum*" sections.
```
Comment (Arielle-Bennett) [inline workshops/book-dash/organisers_files-2022/2022-may-index.md:210]:
```suggestion
    * The [issue template](https://github.com/alan-turing-institute/the-turing-way/blob/main/ISSUE_TEMPLATE.md) will automatically be rendered in the comment section of the new issue page, so all you need to do is edit the "*Lorem ipsum*" sections.
```

File: workshops/book-dash/organisers_files-2022/2022-may-index.md [hunk 14]
```diff
@@ -0,0 +1,275 @@
...
+    * Add the ```book-dash-<location>``` label in the column on the right.
+2. [Fork](https://help.github.com/articles/fork-a-repo) the Turing Way repository to your GitHub profile.
+3. Make your changes!
+4. Submit a [pull request](https://help.github.com/articles/creating-a-pull-request).
+    * Open a pull request as early in your contributing process as possible and add the label [WIP] to the title to designate "work in progress".
+    * The [pull request template](https://github.com/alan-turing-institute/the-turing-way/blob/master/.github/PULL_REQUEST_TEMPLATE.md) will automatically be rendered in the comment section of the new pull request page, so all you need to do is edit the "*Lorem ipsum*" sections.
```
Comment (Arielle-Bennett) [inline workshops/book-dash/organisers_files-2022/2022-may-index.md:216]:
```suggestion
    * The [pull request template](https://github.com/alan-turing-institute/the-turing-way/blob/main/.github/PULL_REQUEST_TEMPLATE.md) will automatically be rendered in the comment section of the new pull request page, so all you need to do is edit the "*Lorem ipsum*" sections.
```

File: workshops/book-dash/organisers_files-2022/2022-nov-index.md [hunk 15]
```diff
@@ -0,0 +1,320 @@
...
+    
+The Turing Way Book Dash committee will co-develop a report providing an overview from the event: https://hackmd.io/@turingway/bookdash-nov2022
+
+:books: _The Turing Way_ links and resources
+---
+
+### Participation guidelines
+
+- Code of Conduct: https://the-turing-way.netlify.app/community-handbook/coc.html
+- Contribution guideline: https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md
```
Comment (Arielle-Bennett) [inline workshops/book-dash/organisers_files-2022/2022-nov-index.md:242]:
```suggestion
- Contribution guideline: https://github.com/alan-turing-institute/the-turing-way/blob/main/CONTRIBUTING.md
```

File: workshops/book-dash/organisers_files-2022/2022-nov-index.md [hunk 16]
```diff
@@ -0,0 +1,320 @@
...
+
+### GitHub Resources
+
+- GitHub repository: https://github.com/alan-turing-institute/the-turing-way
+- Online Book: https://the-turing-way.netlify.app/welcome
+- Impact story: https://www.turing.ac.uk/research/impact-stories/changing-culture-data-science
+    - [Short summary](https://www.turing.ac.uk/sites/default/files/inline-images/Turing%20Way%20oner_0.jpg)
+- Open issues: https://github.com/alan-turing-institute/the-turing-way/issues
+- Open Pull Requests: https://github.com/alan-turing-institute/the-turing-way/pulls
+- [Reports from the previous Book Dashes](https://github.com/alan-turing-institute/the-turing-way/tree/master/workshops/book-dash)
```
Comment (Arielle-Bennett) [inline workshops/book-dash/organisers_files-2022/2022-nov-index.md:264]:
```suggestion
- [Reports from the previous Book Dashes](https://github.com/alan-turing-institute/the-turing-way/tree/main/workshops/book-dash)
```

File: workshops/book-dash/organisers_files-2022/2022-nov-index.md [hunk 17]
```diff
@@ -0,0 +1,320 @@
...
+
+## :gift: Your contribution to the project
+
+**Please read and follow our project [Contributing Guidelines](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md)!** :sparkles:
```
Comment (Arielle-Bennett) [inline workshops/book-dash/organisers_files-2022/2022-nov-index.md:268]:
```suggestion
**Please read and follow our project [Contributing Guidelines](https://github.com/alan-turing-institute/the-turing-way/blob/main/CONTRIBUTING.md)!** :sparkles: 
```

File: workshops/book-dash/organisers_files-2022/2022-nov-index.md [hunk 18]
```diff
@@ -0,0 +1,320 @@
...
+
+**Please read and follow our project [Contributing Guidelines](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md)!** :sparkles: 
+
+If you find them confusing, difficult to follow or think that information needs to be added, please let us know! This will help us improve the contributing experience for everyone going forward.
+
+### Managing your contributions to the GitHub repository
+
+The tl;dr version for easy reference during dashing:
+1. Comment on an [existing issue](https://github.com/alan-turing-institute/the-turing-way/issues) or open a new issue referencing your addition.
+    * The [issue template](https://github.com/alan-turing-institute/the-turing-way/blob/master/ISSUE_TEMPLATE.md) will automatically be rendered in the comment section of the new issue page, so all you need to do is edit the "*Lorem ipsum*" sections.
```
Comment (Arielle-Bennett) [inline workshops/book-dash/organisers_files-2022/2022-nov-index.md:276]:
```suggestion
    * The [issue template](https://github.com/alan-turing-institute/the-turing-way/blob/main/ISSUE_TEMPLATE.md) will automatically be rendered in the comment section of the new issue page, so all you need to do is edit the "*Lorem ipsum*" sections.
```

File: workshops/book-dash/organisers_files-2022/2022-nov-index.md [hunk 19]
```diff
@@ -0,0 +1,320 @@
...
+    * Add the ```book-dash-<location>``` label in the column on the right.
+2. [Fork](https://help.github.com/articles/fork-a-repo) the Turing Way repository to your GitHub profile.
+3. Make your changes!
+4. Submit a [pull request](https://help.github.com/articles/creating-a-pull-request).
+    * Open a pull request as early in your contributing process as possible and add the label [WIP] to the title to designate "work in progress".
+    * The [pull request template](https://github.com/alan-turing-institute/the-turing-way/blob/master/.github/PULL_REQUEST_TEMPLATE.md) will automatically be rendered in the comment section of the new pull request page, so all you need to do is edit the "*Lorem ipsum*" sections.
```
Comment (Arielle-Bennett) [inline workshops/book-dash/organisers_files-2022/2022-nov-index.md:282]:
```suggestion
    * The [pull request template](https://github.com/alan-turing-institute/the-turing-way/blob/main/.github/PULL_REQUEST_TEMPLATE.md) will automatically be rendered in the comment section of the new pull request page, so all you need to do is edit the "*Lorem ipsum*" sections.
```

**Review by Arielle-Bennett — APPROVED** [review]
Added updates to change master to main - keeps links fresh and stops the CI triggering!

**Review by Arielle-Bennett — APPROVED** [review]
Added updates to change master to main - keeps links fresh and stops the CI triggering!

**Comment by malvikasharan** [comment]
Thank you Arielle! 🥇

**Pull request merged**